configTools/src/main/scala/com/gu/janus/model/models.scala

```diff
@@ -42,7 +42,11 @@ object SupportACL {
 }
 
 case class AwsAccount(
+    /* The name of the account as you would like it to appear on Janus */
     name: String,
+    /** The name of the AWS profile to pass in via CLI. Should be relatively
```

Comment:
As mentioned in the PR comment, this string is actually used for three separate purposes. I think it'd be nice if this comment conveyed that information so people are (hopefully) less confused in the future?

Do you feel like the comment on the PR gives you enough context to understand this case class field, and write a comment that conveys that understanding? If not then let me know, and we can look together!

I'd also consider refactoring this into something clearer at some point, but we'll be spending a bit of time on Janus soon so that's probably a good time for us to iron out some of these kinds of kinks.